pom.xml

```diff
@@ -1,351 +1,353 @@
 <project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```

Comment:
Please reduce the diff to only the changes. (Do not format this file since the InteliJ formatter ruins the formatting)

Reply:
Tip: revert this file using `git checkout bb74e16187a5876ef46a91b00ae91304dda90b57 -- pom.xml` and then change the hate version